config/v10/typo3-104.php

```diff
@@ -35,4 +36,16 @@
         ],
     ]]);
     $services->set(UseFileGetContentsForGetUrlRector::class);
+    $services->set('typo3_objectmanagerget_to_generalutilitymakeinstance')
```

Comment:
I'm not sure about a proper naming convention for the services. Maybe you should define a convention upfront, and I'll adjust the PR.